gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -50,7 +50,8 @@
             // Built-in checks
             "ArrayEquals",
             "MissingOverride",
-            "UnnecessaryParentheses");
+            "UnnecessaryParentheses",
+            "PreferJavaTimeOverload");
```

Comment:
So I think the `checkExplicitlyDisabled` function will actually prevent these auto-fixes being applied until we get rid of the explicit `CheckSeverity.OFF` setting... I'm kinda OK with just leaving this here so that when we re-enable the check it will start patching.